src/_cffi_src/openssl/x509.py

```diff
@@ -354,8 +329,8 @@
 /* these functions were added in 1.1.0 */
 const ASN1_INTEGER *X509_REVOKED_get0_serialNumber(const X509_REVOKED *);
 const ASN1_TIME *X509_REVOKED_get0_revocationDate(const X509_REVOKED *);
-void X509_CRL_get0_signature(const X509_CRL *, const ASN1_BIT_STRING **,
-                             const X509_ALGOR **);
+void X509_CRL_get0_signature(const X509_CRL *crl, const ASN1_BIT_STRING **psig,
+                             const X509_ALGOR **palg);
```

Comment:
(This is a mistake in rebase, but I'll fix it when we squash this whole thing for the final PR review)